app/helpers/application_ui_helper.rb

```diff
@@ -0,0 +1,31 @@
+module ApplicationUiHelper
+  # Create a edit button with icon
+  #
+  # @param path [String] The path of the link.
```

Comment:
It also depends on what the local ones are for. If it can be generalised it should be made global.